packages/rancher-vsphere-cpi/charts/templates/vsphere-cpi-ds.yaml

```diff
@@ -1,11 +1,11 @@
-# Source: https://github.com/kubernetes/cloud-provider-vsphere/blob/master/releases/v1.19/vsphere-cloud-controller-manager.yaml
+# Source: https://github.com/kubernetes/cloud-provider-vsphere/blob/master/releases/v1.19/{{ .Chart.Name }}-cloud-controller-manager.yaml
```

Comment:
this is unnecessary 😆 
```suggestion
# Source: https://github.com/kubernetes/cloud-provider-vsphere/blob/master/releases/v1.19/vsphere-cloud-controller-manager.yaml
```

Reply:
Good catch, did a find and replace and missed that.